### PR TITLE
Devs can add multiple PR links to Dev Portfolio Assignment Submission 

### PR DESCRIPTION
**Pull request by airaagrawal1**
### Summary 
Implemented functionality for devs to add fields to submit more than 1 opened/reviewed pull request for a dev portfolio assignment. 

### Test Plan 
Manually tested buttons and made sure correct fields were being added/deleted. Submission button returns an error as of now because this has not been connected to the backed.

Below is a screenshot of the current UI:
Screen Shot 2022-05-06 at 8 49 22 PM

### CODE REVIEW

File: frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx [hunk 1]
```diff
@@ -69,22 +70,22 @@ const DevPortfolioForm: React.FC = () => {
         headerMsg: 'No Dev Portfolio selected',
         contentMsg: 'Please select a dev portfolio assignment!'
       });
-    } else if (!openPR || !reviewedPR) {
+    } else if (!openPRs[0] || !reviewPRs[0]) {
```
Comment (henry-li-06) [inline frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx:73]: maybe should check to make sure that none of the inputs are `''`

**Comment by henry-li-06** [comment]
Should we add regex to make sure that the submitted links are valid links to GitHub PRs?

**Review by riyajaggi — COMMENTED** [review]
Great work on implementing this PR! I have added some minor comments about code style, other than that it looks good.

**Comment by riyajaggi** [comment]
> Should we add regex to make sure that the submitted links are valid links to GitHub PRs?

I like this idea! It would definitely make it more robust and avoid unnecessarily errors when reviewing the PRs. I recommend trying to find a pre-existing package that could validate links since regex could be long and require future maintenance, but it'll probably be 2-3 lines if you do have to do it manually.

**Comment by dan-ieljin** [comment]
This form looks great overall! One minor comment: under each link input, the "X"s to delete that link might be more intuitive when placed on the right side of the text input, especially if there are several links given and they start to get a bit bundled together.

**Comment by henry-li-06** [comment]
> This form looks great overall! One minor comment: under each link input, the "X"s to delete that link might be more intuitive when placed on the right side of the text input, especially if there are several links given and they start to get a bit bundled together.

Yeah that's a good point, having the "X" on the side might be better

**Comment by JacksonStaniec** [comment]
> > Should we add regex to make sure that the submitted links are valid links to GitHub PRs?
> 
> I like this idea! It would definitely make it more robust and avoid unnecessarily errors when reviewing the PRs. I recommend trying to find a pre-existing package that could validate links since regex could be long and require future maintenance, but it'll probably be 2-3 lines if you do have to do it manually.

The function `parseGithubUrl()` in the `githubUtil.ts` file in the backend has an example of a regex for github PR links if you want. Since we need to extract information like owner, repo, and pull number from the URL, we might need to maintain our own regex for the future anyways. Unless theres an existing package that does that anyways, which would be the best for everyone!

**Review by JacksonStaniec — APPROVED** [review]
```
        _    _
     __|_|__|_|__
   _|____________|__
  |o o o o o o o o /  
~'`~'`~'`~'`~'`~'`~'`~
```

**Pull request merged**